src/javascript/crypto/e2e/extension/constants.js

```diff
@@ -23,6 +23,7 @@ goog.provide('e2e.ext.constants.Actions');
 goog.provide('e2e.ext.constants.BackupCode');
```

Comment:
The file isn't needed - I'm not sure why it's being shown here, my branch isn't tracking it. 
I'll look into this.